contrib/completion/bash/docker

```diff
@@ -5485,6 +5485,23 @@ _docker_wait() {
 	_docker_container_wait
 }
 
+_docker_compose() {
+	local composePluginPath="${HOME}/.docker/cli-plugins/docker-compose"
```

Comment:
This location would only work (by default) on Docker Desktop; on Linux, plugins may be installed elsewhere.

```bash
docker info --format '{{json .ClientInfo.Plugins}}' | jq .
[
  {
    "SchemaVersion": "0.1.0",
    "Vendor": "Docker Inc.",
    "Version": "v0.9.1-beta3",
    "ShortDescription": "Docker App",
    "Experimental": true,
    "Name": "app",
    "Path": "/usr/libexec/docker/cli-plugins/docker-app"
  },
  {
    "SchemaVersion": "0.1.0",
    "Vendor": "Docker Inc.",
    "Version": "v0.5.1-docker",
    "ShortDescription": "Build with BuildKit",
    "Name": "buildx",
    "Path": "/usr/libexec/docker/cli-plugins/docker-buildx"
  },
  {
    "SchemaVersion": "0.1.0",
    "Vendor": "Docker Inc.",
    "Version": "v0.7.0",
    "ShortDescription": "Docker Scan",
    "Name": "scan",
    "Path": "/usr/libexec/docker/cli-plugins/docker-scan"
  }
]
```

Reply:
I should not that if we _do_ want to use  the above `docker info` to get that information, and we must do that on every request, we should have a look if we can somehow get the info without the CLI also making an API call (to the `/info` endpoint) 😬